### PR TITLE
Added button variables

### PR DESCRIPTION
**Pull request by bryce-seifert**

### CODE REVIEW

File: HELP.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Module for Extron SMP 351
 
-> this module connects via a telnet connection.
+This module connects to the Extron H.264 Streaming Media Processor (SMP) 351 and allows you to login, send commands, and recieve feedback based on the status of the device.
```
Comment (EventReplay) [inline HELP.md]: Leave "> " in, It's formatting, purely cosmetic, I think it looks nicer.

File: index.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ instance.prototype.init = function() {
 
 	self.states = {}
 	self.init_feedbacks()
-
```
Comment (EventReplay) [inline index.js:36]: This is my mistake but I thik these should have a ; after them.

File: index.js [hunk 3]
```diff
@@ -71,18 +75,35 @@ instance.prototype.incomingData = function(data) {
 		self.login = false;
 		self.status(self.STATUS_WARNING,'Checking Connection');
 		self.socket.write("2I"+ "\n"); // should respond with model description eg: "Streaming Media Processor"
+		self.socket.write("36I"+ "\n");
```
Comment (EventReplay) [inline index.js]: self.socket.write("36I"+ "\n");
This dosn't need to be here, the response from 2I will trigger data.match and run it.

File: HELP.md (hunk 1 above)
Comment (bryce-seifert) [reply HELP.md]: Got it, re-added.

File: index.js (hunk 2 above)
Comment (bryce-seifert) [reply index.js:36]: I was curious about that, fixed.

File: index.js (hunk 3 above)
Comment (bryce-seifert) [reply index.js]: Noted and removed.

**Pull request merged**